outdated/modules/httpd.reb

```diff
@@ -112,29 +127,26 @@ sys/make-scheme [
                 spec/port-id: spec/ref/3
                 spec/actions: spec/ref/4
             ]
-            /else [
-                do make error! "Server lacking core features."
-            ]
+            default [fail "Server lacking core features."]
         ]
 
         server/locals: make object! [
-            handler: subport: open?: _
```

Comment:
Stylistic change for module repository. Please put unrelated initializations on their own lines. It's easier to scan such lists and more "transparent".

Also when such lists need to be changed...e.g. to assign to one element which had been somewhere in the middle--the diff will be clearer.